src/main/resources/db/migration/courtcase/V2133__remove_duplicate_recrods.sql

```diff
@@ -0,0 +1,31 @@
+BEGIN;
+
+DROP TABLE IF EXISTS court_case_new;
+
+create table court_case_new as select * from court_case cc where cc.id  in (select max(id) from court_case cc2 group by cc2.case_id);
+
+ALTER TABLE court_case_new ADD PRIMARY KEY (id);
+
```

Comment:
Add this here to auto generate ID
```suggestion

ALTER TABLE court_case_new
ALTER COLUMN id
ADD GENERATED ALWAYS AS IDENTITY;
```